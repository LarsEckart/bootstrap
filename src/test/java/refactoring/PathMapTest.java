package refactoring;

import org.approvaltests.Approvals;
import org.approvaltests.reporters.UseReporter;
import org.approvaltests.reporters.macosx.BeyondCompareMacReporter;
import org.junit.jupiter.api.Test;

@UseReporter(BeyondCompareMacReporter.class)
class PathMapTest {

  @Test
  void test_initialization() {
    Approvals.verify(XMLToJson.pathMap);
  }
}
