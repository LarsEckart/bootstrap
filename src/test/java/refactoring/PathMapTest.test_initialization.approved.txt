dk : doc[@key 
dt : doc[@type 
dth : doc[@type='history' 
dtrn : doc[@trnum 
fk : folder[@key 
ft : folder[@type 
fth : folder[@type='history' 
